app/jobs/solidus_subscriptions/process_subscription_and_installment_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module SolidusSubscriptions
+  class ProcessSubscriptionAndInstallmentJob < ApplicationJob
```

Comment:
I would probably call this `ProcessSubscriptionJob`, on a second thought. It doesn't really process the installment, it just schedules it for processing. Calling it `ProcessSubscriptionAndInstallmentJob` implies that this also does the job of `ProcessInstallmentJob`.

Reply:
That makes sense, I'm renaming it.